akka-actor-typed/src/main/scala/akka/actor/typed/internal/eventstream/SystemEventStream.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.internal.eventstream
+
+import akka.actor.typed.Behavior
+import akka.actor.typed.eventstream.EventStream
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.adapter._
+import akka.annotation.InternalApi
+
+/**
+ * INTERNAL API
+ * Encapsulates the [[akka.actor.ActorSystem.eventStream]] in a [[Behavior]]
+ */
+@InternalApi private[akka] object SystemEventStream {
```

Comment:
I think it would be better to put it with all the other adaptors of untyped things into typed in the package `akka.typed.internal.adapter`, and maybe also call it `EventStreamAdapter`.

Reply:
Just a move?

Reply:
I'm not an akka regular, I followed this to:

1. Highlight that it's the actor system event stream
2. It's a wrapper against the untyped event stream

As in my mind, I imagine this will change by providing a full typed implementation of the EventBus at some point in the future. What I wanted to achieve is to make the life easier of the person that's coming back in the future to this .

If what you suggest is the convention then I imagine that would be the best thing to do.

Reply:
Wasn't a bad place to start but since all the other adapting things are there that's where the surface area against untyped is and where you'd start looking, so moving sounds good!